tokens/src/themes/light/components/Alert.json

```diff
@@ -21,6 +21,20 @@
         "info": { "source": "$alert-info-border-color", "$value": "{color.theme.border.info}" },
         "danger": { "source": "$alert-danger-border-color", "$value": "{color.theme.border.danger}" },
         "warning": { "source": "$alert-warning-border-color", "$value": "{color.theme.border.warning}" }
+      },
+      "actions": {
+        "overrides": {
+          "button": {
+            "variants": {}
```

Comment:
[inform] The token schema exists, but is a no-op for default (i.e., `variants` is empty object, so the style-dictionary format won't have any key/values to iterate over by default. The `examples` key below intends to act as in-line documentation around what to pass into `variants`, though the primary documentation for this schema will likely live in `brand-openedx` (PR still to come).